src/GovUk.Education.ExploreEducationStatistics.Public.Data.Processor/Repository/FilterMetaRepository.cs

```diff
@@ -126,6 +132,10 @@ await optionTable
                     $"Inserted incorrect number of filter option meta links for {meta.PublicId}. " +
                     $"Inserted: {insertedLinks}, expected: {options.Count}");
             }
+
+            await publicDataDbContext.Database.ExecuteSqlAsync(
+                $"SELECT setval('public.\"FilterOptionMetaLink_Id_seq\"', {startIndex + insertedLinks - 1})",
+                cancellationToken);
```

Comment:
Similar to earlier comment, might be good to abstract this into a method on `PublicDataDbContext`:

```cs
public Task SetSequenceValue(string sequenceName, long value, CancellationToken cancellationToken = default)
    => Database.ExecuteSqlAsync(
        $"SELECT setval('public.\"{sequenceName}\"', {value})",
        cancellationToken);
```

This can be re-used for importing location options and the seeder